core/views.py

```diff
@@ -113,30 +114,30 @@ def get_context_data(self, **kwargs):
                             output_field=IntegerField(),
                         )
                     ),
-                    sum_empty_net=Sum(
+                    sum_goals_against=Sum(
                         Case(
                             When(stat__team=F('roster__team'), stat__team__is_active=True,
-                                    then=Coalesce('stat__empty_net', 0)),
+                                    then=Coalesce('stat__goals_against', 0)-Coalesce('stat__empty_net', 0)),
```

Comment:
We should confirm with Wayne, but I don't think he adds a goal against and an empty net when inputting stats. Example: if the final score was 5-1 with one empty net, I think Wayne enters 4 goals against, and then 1 empty net. We'd definitely want to make sure though.